translations/strings.yaml

```
@@ -443,10 +443,81 @@ mobileOnboarding:
+  productUpdateOptInDescription: 
+    value: I’d like to receive product update emails from Firefox.
+    comment: User has the option to opt-in to receive product update emails from Firefox. They are asked this when creating an account with us. 
+  createAccountButton: Create account
+  finishAccountCreationDescription: 
+    value: Finish creating your Firefox account to continue to Mozilla VPN
+    comment: User message prompting user to finish creating their account to use Mozilla VPN. 
+  emailVerificationDescription: 
+    value: Open your email and enter the verification code it that was sent.
+    comment: User has been sent a verification email containing a code. 
+  invalidPasswordErrorMessage: 
+    value: Invalid password. Please try again
+    comment: User has submitted an invalid password.
+  invalidCodeErrorMessage: 
+    value: Invalid code. Please try again
+    comment: User provided did not provide a correct security code.
+  invalidEmailErrorMessage: 
+    value: Invalid email. Please try again
+    comment: Generic message for when user submits incorrect email.
+  invalidEmailFormatErrorMessage: 
+    value: Email format not supported. Please try again
+    comment: User is providing us an email that is not formatted correctly. Ex. jsmith&gmail.com
+  # Verification code is used when user is creating a new account.  
+  verificationCodeTitle: Enter verification code
+  verificationCodeInputPlaceholder: 
+    value: Enter 6-digit code
+    comment: Placeholder text for input element asking user for 6-digit verification code from their email.
+  verifySecurityCodeButton: Verify
+  resendCodeLink: 
+    value: Resend code
+    comment: This link allows the user to request for another 6-digit verification code to be sent to their email.
+  # Security code is used for 2FA purposes.   
+  securityCodeLabel: 
+    value: Security code
+    comment: Label for input asking user for their security code. 
+  securityCodeTitle: 
+    value: Enter security code  
+    comment: Title for app screen asking user for their security code.
+  securityCodeSubtitle: 
+    value: Open your authentication app and enter the security code it provides.
+    comment: User is asked for a security code from their authentication app for verification.
+  informationUsageDisclaimer: 
+    value: We only use your email to create your account or sign in. We will never sell it to a third party.
+    comment: Non-op text that informs user about how Mozilla will not sell it to a third party. 
+  signInFailedPopupTitle: Sign-in failed
+  signInFailedPopupDescription: 
+    value: You’ve made too many attempts to sign in. You won’t be able to try again with this account for 15 minutes.
+    comment: User has been unable to sign in correctly, they have a cool down period of 15 minutes until they can try again.
+  problemSigningInTitle: 
+    value: Problem signing in...
```

```suggestion
    value: Problem signing in…
```

We have a [linter](https://github.com/mozilla-mobile/mozilla-vpn-client/runs/5531456469?check_suite_focus=true) for this ;-)